gradle/libs.versions.toml

```diff
@@ -24,6 +24,7 @@ mockKVersion = "1.13.10"
 targetSdk = "34"
 timberVersion = "5.0.1"
 truth = "1.4.2"
+material = "1.12.0"
```

Comment:
I had already thought about whether it might make sense in the long term to develop an android studio plugin that supports you with things like creating new modules, etc. and also helps you to create new ViewModels and Views with live templates to make it easier following along naming convention, etc.

but also helps you to create a new application with the spezi framework. e.g. you can select a certain module and then automatically get the classes created that have to be implemented so that the module can be used... but i think that would be something more long-term.

Reply:
Thanks @eldcn + @Basler182 for your thoughts! Your rationale makes a lot of sense. Will let @PSchmiedmayer weigh in here also before making a decision.